Makefile

```diff
@@ -84,6 +84,9 @@ include contrib/devtools/Makefile
 ###############################################################################
 
 build: go.sum
+	go build -mod=readonly ./...
+
+install: go.sum
```

Comment:
Both `build` and `install` do the same thing - plus `install` installs nothing in fact. Why keeping both?

Reply:
I found it confusing that build installed something. It was unexpected so I added an install